djstripe/models.py

```diff
@@ -841,6 +876,7 @@ def log(cls, data, exception, event):
         )
 
     def __str__(self):
+        """Return string repr of EventProcessingException."""
```

Comment:
Also see if it's possible to ignore nonexistant docs on all builtin methods (``__<method_name>__()``)